src/util.js

```diff
@@ -406,19 +408,32 @@
    */
   connect.PopupManager = function () { };
 
-  connect.PopupManager.prototype.open = function(url, name) {
+  connect.PopupManager.prototype.open = function (url, name, options) {
     var then = this._getLastOpenedTimestamp(name);
     var now = new Date().getTime();
-    var win = null;      
+    var win = null;
     if (now - then > ONE_DAY_MILLIS) {
-       win = window.open('', name);
-       if (win.location !== url) {
+      if (options) {
+        // default values are chosen to provide a minimum height without scrolling
+        // and a uniform margin based on the css of the ccp login page
+        var height = options.height ? options.height : DEFAULT_POPUP_HEIGHT;
+        var width = options.width ? options.width : DEFAULT_POPUP_WIDTH;
+        var top = isNaN(options.top) ? 0 : options.top;
+        var left = isNaN(options.left) ? 0 : options.left;
```

Comment:
Not sure if you are done yet but the 'canonical' way to do these (there are examples in other areas in this project as well) is by using javascript's short cut as the elvis operator.
For example
```
const options = {}
var popupTop = options.top || 0
console.log(popupTop)
options.top = 10
popupTop = options.top || 0
console.log(popupTop)
```

Reply:
updated to

```js
var height = options.height || DEFAULT_POPUP_HEIGHT;
var width = options.width || DEFAULT_POPUP_WIDTH;
var top = options.top || 0;
var left = options.left || 0;
```